target/linux/mvebu/profiles/000-Default.mk

```diff
@@ -8,7 +8,8 @@
 define Profile/Default
 	NAME:=Default Profile (all drivers)
 	PACKAGES:= kmod-mwlwifi wpad-mini swconfig mkf2fs e2fsprogs \
-		   mwlwifi-firmware-88w8864 mwlwifi-firmware-88w8964
+		   mwlwifi-firmware-88w8864 mwlwifi-firmware-88w8964 \
+		   mwlwifi-firmware-88w8997
```

Comment:
If this FW is not used on any board, please do not add it to the default packages.